force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -159,9 +189,9 @@ public with sharing class VariablesDataTypesOperators {
      * @return The number of milliseconds equivalent to the given number of minutes.
      */
     public static Integer convertMinutesToMilliseconds(Integer minutes) {
-        Integer MILLISECONDS_PER_MINUTE = null; // Make this value a constant
-        Integer milliseconds;
-        return null; // Replace null with the variable you used to store the result
+        Integer MILLISECONDS_PER_MINUTE = minutes / 10000;// Make this value a constant
+        Integer milliseconds = minutes * 60000;
+        return milliseconds; // Replace null with the variable you used to store the result
```

Comment:
This is not correct. A constant should never change its value. In  this case milliseconds per minute is being assigned a new value every time the method is called.  But you are returning the correct answer just not using the constant.